stack-persistent-213.yaml

```diff
@@ -0,0 +1,22 @@
+resolver: nightly-2021-03-31
```

Comment:
Can you add this config to the CI build matrix here? https://github.com/yesodweb/yesod/blob/master/.github/workflows/tests.yml#L24